review/fileToReview.js

```diff
@@ -1,5 +1,10 @@
-
-var domain = "bank.local.fr"
```

Comment:
use 'const' instead of 'var'.  Var have a biggest litter than const and should be avoided if possible. (use const / let instead)

Same for the next code

I have also put it inside a config object for more clarity since i have also merged common headers in this object.

Next step will be to set them in ENV vars for a better management